src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -255,6 +248,12 @@ export function MapAnalysisImpl(props: Props & CompleteAppState) {
     studyMetadata.rootEntity.id,
   ]);
 
+  const totalEntitiesInSampleCount = (() => {
+    if (!totalCounts.value || !outputEntity) return 0;
+
+    return totalCounts.value[outputEntity.id];
+  })();
```

Comment:
What is the reason for the iife here? It seems unnecessary. Also, I think this variable should be named something like `outputEntityTotalCount`. "Samples" is just one possible entity type.

Reply:
Dunno, it looked good to me on Friday afternoon. I agree with you though. I changed it to in this commit: https://github.com/VEuPathDB/web-eda/pull/1656/commits/496ae2e57394b4ecf324ae772f939b0e7926b7c9.

```js
  const outputEntityTotalCount = totalCounts.value && outputEntity ? totalCounts.value[outputEntity.id] : 0;
```